Nanoleaf-wpf/Views/App.xaml.cs

```diff
@@ -52,6 +56,8 @@ void App_Startup(object sender, StartupEventArgs e)
             {
                 NormalStartup(e);
             }
+
+            CheckForUpdate();
```

Comment:
CheckForUpdate should be called at the end of NormalStartup instead of here

Reply:
Fixed in update